feedparser/http.py

```diff
@@ -108,7 +108,7 @@ def _build_urllib2_request(url, agent, accept_header, etag, modified, referrer,
         request.add_header('Referer', referrer)
     request.add_header('Accept-encoding', 'gzip, deflate')
     if auth:
-        request.add_header('Authorization', 'Basic %s' % auth)
+        request.add_header('Authorization', 'Basic %s' % str(auth, 'utf8'))
```

Comment:
This old style substitution can be converted into f-string.

Reply:
This change is no longer required now that auth isn't a byte string.